doc/classes/AudioEffectCompressor.xml

```diff
@@ -110,18 +118,25 @@
 	</methods>
 	<members>
 		<member name="attack_us" type="float" setter="set_attack_us" getter="get_attack_us">
+			Time it takes for the compressor to react when the signal exceeds the threshold. Value can range from 20 to 2000. Default value: [code]20ms[/code].
 		</member>
 		<member name="gain" type="float" setter="set_gain" getter="get_gain">
+			Amount of gain applied to the output signal.
```

Comment:
"Amount of gain" is redundant, since gain is an amount. Just use "Gain applied..."